notebooks/text-classification_multilabel.ipynb

```diff
@@ -0,0 +1,548 @@
+{
```

Comment:
<p>Could we uncomment this line?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/setfit/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>